settings.gradle

```diff
@@ -5,7 +5,7 @@ pluginManagement {
         maven {
             url = "https://repo.screamingsandals.org/public/"
         }
-
+        maven { url "https://maven.mrnavastar.rocks/releases" }
```

Comment:
shouldn't be defined here, not required by any gradle plugin

Reply:
Resolved in commit https://github.com/ScreamingSandals/BedWars/pull/539/commits/a732f4d6e903ff6858830dd363f1f72bb9dc93bc